### ISSUE TITLE
WindowsPhone-Support

### ISSUE THREAD

**Issue by dlangerenken** [Integreat/app-cross_platform#62]
We should support windows phone now

**Comment by mschrimpf** [comment]
Shouldn't be our highest prio though, according to the statistics we have only very few refugees use Windows phones.

**Comment by dlangerenken** [comment]
It's more about listing it as a feature we have ;)

**Comment by pbednarek** [comment]
I'm leaving behind the following unresolved issues:

**Critical (needs to be done before release)**

- “Disclaimer” shuts down app (NavigationViewModel.OnOpenDisclaimerClicked throws ArgumentNullException)

**Non-Critical (nice to have before release)**

- Special characters in texts are not displayed correctly (for example the “it`s” becomes “it&#8217;s”)
- Symbols in Event texts too large – container “imageCell” does not allow for size manipulation
- “…” in top bar without functionality – Maybe there’s a way to hide it?
- App shows a margin at the top on the testing device – couldn’t find out how to hide it

**Comment by BerndNK** [comment]
Postponed until further notice.
